roxie/ccd/ccdmain.cpp

```diff
@@ -1513,18 +1517,20 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
                         StringBuffer certFileName;
                         StringBuffer keyFileName;
                         StringBuffer passPhraseStr;
-                        Owned<IPropertyTree> tlsConfig;
+                        Owned<const ISyncedPropertyTree> tlsConfig;
                         if (serviceTLS)
                         {
                             protocol = "ssl";
 #ifdef _USE_OPENSSL
                             if (isContainerized())
                             {
+                                //MORE: Why is this containerized only?  Should it check for this in both, and fall back to the
+                                //old config if none found and non-containerized?
```

Comment:
Yes, absolutely agreed, that was the ultimate goal.  Bare-metal and fusion should be able to treat certificate config as issuer secrets if they want, and if that isn't present fall back to the old style config.

Reply:
created hpcc-30754